### ISSUE TITLE
nexrender-worker freeze

### ISSUE THREAD

**Issue by zuntah** [inlife/nexrender#246]
**Describe the bug**
I have a nexrender-server and 2 workers which are connected and pulling jobs every 30 seconds (Default values).
The problem is that sometimes the workers freeze and don't take new jobs, once you go to the CMD windows and click ENTER suddenly the job is being received and the worker is starting to process it.

**Information about environment**
* Which version of nexrender are you using?
1.8.6
* What kind of the setup do you have? (server+worker or cli)
Server (Linux VPS in DigitalOcena)
2 Workers (Win10 Machine)
* Are you using pre-built binaries or globally installed npm packages?
The server is running from source, workers from NPM -g
* Which kind (and version) of operating system you are running it in?
Server - Ubuntu 18, Workers - Win10
* Any additional information that might be helpful

**Expected behavior**
The workers should not freeze and all the time receive new jobs from the server.

**Comment by inlife** [comment]
Please check if it behaves similarly to this case
https://github.com/inlife/nexrender/issues/191#issuecomment-512980182

**Comment by zuntah** [comment]
Yes, Similar behavior.
What is the solution to this problem? Maybe running with Powershell?

**Comment by inlife** [comment]
The cause for that problem and solution for it was described in the 2nd following comment:
https://github.com/inlife/nexrender/issues/191#issuecomment-512986828

TLDR: user selected some data in the console, causing the whole process to freeze

**Comment by zuntah** [comment]
Got it, but I'm not sure 100% that it was the reason (The text marking).
I am quite sure that it's the windows shell because I don't have this issue with nexrender-server which is being executed on Linux cloud server.

Anyways, I thought of a better solution.
I think I will just use pm2 in order to run the worker from source code.
That's also a good way to keep it updated.

What do you think? 
Thanks.

**Comment by inlife** [comment]
I guess yea, you are right. That solution might be quite viable.

**Comment by zuntah** [comment]
Solved by running with pm2.
Thanks :)
You can close the issue.
